contrib/fish-shell/template.py

```diff
@@ -0,0 +1,17 @@
+pkgname = "fish-shell"
+pkgver = "3.5.1"
+pkgrel = 0
+build_style = "cmake"
+hostmakedepends = ["cmake", "ninja", "pkgconf", "gettext-tiny"]
+makedepends = ["ncurses-devel", "pcre2-devel"]
+pkgdesc = "Friendly interactive command line shell"
+maintainer = "lunamothxyz <mail@lunamoth.xyz>"
+license = "GPL-2.0-only"
+url = "https://fishshell.com"
+source = f"https://github.com/fish-shell/{pkgname}/releases/download/{pkgver}/fish-{pkgver}.tar.xz"
+sha256 = "a6d45b3dc5a45dd31772e7f8dfdfecabc063986e8f67d60bd7ca60cc81db6928"
+# incompatible with cbuild - fish's checks only work as non-root user
```

Comment:
what does this mean? cbuild does not run checks as root

Reply:
ok, sorry, i'll change the wording. i assumed it fails for the same reason as under xpbs-src. regardless of the reason, it does not like the chroot. the checks pass when building outside of cbuild

Reply:
post the logs, maybe?

Reply:
at least on `ppc64le`, all tests pass with `checkdepends = ["python", "procps-ng"]`

Reply:
[log.txt](https://github.com/chimera-linux/cports/files/9660264/log.txt)

Reply:
if you read the details,

```
 72/236 Test  #84: bad-option.fish ..........................***Failed    0.05 sec
/builddir/fish-shell-3.5.1/build/tests/test.fish (line 37): The expanded command was empty.
    $python -S ../littlecheck.py \
        --progress $force_color \
        -s fish=../test/root/bin/fish \
        -s fish_test_helper=../test/root/bin/fish_test_helper \
        $files_to_test
```

pretty much just indicates that python was not found, which is used by the specific test

nothing about root there